wormhole-connect/src/utils/index.ts

```diff
@@ -103,8 +103,12 @@ export function getTokenById(tokenId: TokenId): TokenConfig | undefined {
   );
 }
 
-export function getDisplayName(token: TokenConfig) {
-  return token.displayName || token.symbol;
+export function getDisplayName(token: TokenConfig, chain: Chain): string {
+  let displayName = token.displayName ?? token.symbol;
+  if (chain !== token.nativeChain) {
+    displayName = `${'Wormhole-wrapped'} ${displayName}`;
```

Comment:
petty nit but you dont need to wrap that first bit
```suggestion
    displayName = `Wormhole-wrapped ${displayName}`;
```

Reply:
changed